TextInputKit/TextInputKit/Code/SpecializedTextInput/CardNumber/BankCardNumberUtils.swift

```diff
@@ -0,0 +1,283 @@
+//
+//  BankCardNumberUtils.swift
+//  TextInputKit
+//
+//  Created by Artem Starosvetskiy on 30/11/2016.
+//  Copyright © 2016 Artem Starosvetskiy. All rights reserved.
+//
+
+import Foundation
+
+struct BankCardNumberUtils {
+
+    struct IinRangeInfo {
+
+        let range: CountableClosedRange<Int>
+
+        let cardBrand: BankCardBrand
+
+        let maxCardNumberLength: Int
+
+        static func info(_ range: CountableClosedRange<Int>, _ cardBrand: BankCardBrand, _ maxCardNumberLength: Int) -> IinRangeInfo {
```

Comment:
This method is used to shorten the following `private static let iinRangesInfo: [IinRangeInfo]` initializer.